coursebook/modules/m3/hands-on.ipynb

```diff
@@ -2,9 +2,34 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "# Hands-on"
+    "## Hands-on Session\n",
+    "\n",
+    "Practice building figures.\n",
+    "\n",
+    "Explore the world in data dataset. You are free to use the full dataset (`df`), which has entries for many dates, or the aggregated dataset which only have the totals (`df_totals`).\n",
```

Comment:
Do we want to use the whole hands-on session for this or should we ask them to explore the EQLS dataset in this time, to continue the hands-on session flow? I think they could be asked to do what currently happens in 3.5 (with guiding questions maybe) and we can use the 3.5 stuff as a guide for instructors?